homeassistant/components/device_tracker/unifi.py

```diff
@@ -41,6 +41,8 @@
         cv.boolean, cv.isfile),
     vol.Optional(CONF_DETECTION_TIME, default=DEFAULT_DETECTION_TIME): vol.All(
         cv.time_period, cv.positive_timedelta),
+    vol.Optional(CONF_MONITORED_CONDITIONS):
+        vol.All(cv.ensure_list, [cv.string]),
```

Comment:
There should be a list of accepted conditions that we validate against here. Please look at sensor platforms that use monitored conditions for examples.

Reply:
Added a list of available attributes as requested